coderd/database/db.go

```diff
@@ -13,9 +13,12 @@ import (
 	"database/sql"
 	"errors"
 
+	"github.com/google/uuid"
 	"golang.org/x/xerrors"
 )
 
+var SystemUserID uuid.UUID = uuid.MustParse("c0de2b07-0000-4000-A000-000000000000")
```

Comment:
Let's add a comment here. Why was this UUID chosen? Why not `uuid.Nil`?

Reply:
`uuid.Nil` already has meaning in the db. We use it as a `null` in search filters since sqlc didn't support nullable args at the time. (It does as of like a week ago, I made a bug report on it as I ran into some issues using `sqlc.narg`)